### ISSUE TITLE
Any plans for a new release?

### ISSUE THREAD

**Issue by rotexdegba** [brandonsavage/Upload#113]
I am currently using this package in a project and I had to specify **dev-master** in my composer file because there are some classes that have been added that are not part of the latest release. I am not comfortable specifying dev-master in my composer file and would really love if a new stable release could be created ASAP.

Thanks for this wonderful package.

**Comment by DavidWiesner** [comment]
Yes please tag a new release so you can get the *latest* (two years old) changes via packagist.

**Comment by iRaziul** [comment]
I don't think so :(

**Comment by saschanos** [comment]
@brandonsavage, would it be possible to give more developers access to the project so we can do some fixes here (e.g. bringing the library to php 8.0)?

**Comment by jakejackson1** [comment]
We've forked the library over at https://github.com/GravityPDF/Upload and have made a number of improvements, including PHP8.2 support. Happy to accept any PRs to continue improving the library.

**Comment by rotexdegba** [comment]
> We've forked the library over at https://github.com/GravityPDF/Upload and have made a number of improvements, including PHP8.2 support. Happy to accept any PRs to continue improving the library.

Thanks
